shortReadsqc.wdl

```diff
@@ -0,0 +1,283 @@
+# Short reads QC workflow
+version 1.0
+
+workflow ShortReadsQC {
+    input{
+        String  container="bfoster1/img-omics:0.1.9"
+        String  bbtools_container="microbiomedata/bbtools:38.96"
+        String  workflow_container = "microbiomedata/workflowmeta:1.1.1"
+        String  proj
+        String prefix=sub(proj, ":", "_")
+        Array[String] input_files
+        String  database="/refdata/"
+    }
+
+    if (length(input_files) > 1) {
+        call stage_interleave {
+        input:
+            container=bbtools_container,
+            memory="10G",
+            input_fastq1=input_files[0],
+            input_fastq2=input_files[1]
+        }
+    }
+    if (length(input_files) == 1) {
+        call stage_single {
+        input:
+            container=container,
+            input_file = input_files[0]
+        }
+    }
+
+    # Estimate RQC runtime at an hour per compress GB
+   call rqcfilter as qc {
+        input:
+            input_fastq = if length(input_files) > 1 then stage_interleave.reads_fastq else stage_single.reads_fastq,
+            threads = "16",
+            database = database,
+            memory = "60G",
+            container = bbtools_container
+    }
+    call make_info_file {
+        input: info_file = qc.info_file,
+            container = container,
+            prefix = prefix
+    }
+
+    call finish_rqc {
+        input: container = workflow_container,
+            prefix = prefix,
+            filtered = qc.filtered,
+            filtered_stats = qc.stat,
+            filtered_stats2 = qc.stat2
+    }
+    output {
+        File filtered_final = finish_rqc.filtered_final
+        File filtered_stats_final = finish_rqc.filtered_stats_final
+        File filtered_stats2_final = finish_rqc.filtered_stats2_final
+        File rqc_info = make_info_file.rqc_info
+    }
+}
+
+
+task stage_single {
+    input{
+        String container
+        String target="raw.fastq.gz"
+        String input_file
+    }
+   command <<<
+
+    set -e
+    if [ $( echo ~{input_file}|egrep -c "https*:") -gt 0 ] ; then
+        wget ~{input_file} -O ~{target}
+    else
+        ln ~{input_file} ~{target} || cp ~{input_file} ~{target}
```

Comment:
remove hard link. out of scope for this review but before we run on tahoma we need update to listing input files properly so they are fetched by the head node instead of compute nodes.